### ISSUE TITLE
Set timezone in the settings

### ISSUE THREAD

**Issue by follower46** [clockworkpi/launcher#53]
It would be nice to have the option to set the system timezone in the settings menu.
I copied the `/usr/share/zoneinfo/America/Chicago` file to `/etc/localtime` on my GS and it updated accordingly.

We could have a vertical list of string options by listing the options in `zoneinfo` and drilling down to the timezone of each location.

I'd like to work on this one :smile:

**Comment by cuu** [comment]
cool,go ahead :)

**Comment by MrP1xel** [comment]
Yes, it will be a good thing to be able to set up the time parameters from the UI
I would like to work on it too :)

**Comment by follower46** [comment]
Hmmmm, I have the list pulling up all the options but when I try to set one I get a permission error.
I ended up using an elevated subprocess call to get around the permissions error but I'm not sure if that's the best way.
https://www.youtube.com/watch?v=v95yVLmdNAQ

Any tips, @cuu?

**Comment by follower46** [comment]
I'm also going to have it highlight the currently set timezone (GMT if none is detected).

**Comment by follower46** [comment]
I also changed the location scanning directory to `/usr/share/zoneinfo/posix` instead of just `/usr/share/zoneinfo` as there was a bunch of weird extra junk in that directory

**Comment by follower46** [comment]
Please let me know what I can change:
https://github.com/clockworkpi/launcher/pull/60

**Comment by cuu** [comment]
If I am right, user cpi on GS  has  sudo power without password
and I reset the code because you pull is not working right now
I'll wait you to finish 
:)

**Comment by follower46** [comment]
Interesting, it wasn't working?
Hmmmm, let me double check. I re-imaged my GS last night and put this on it and it appeared to work.

I'll merge master in and see if I missed something.

**Comment by follower46** [comment]
Ack! I just pulled in d8807f938688de0bfda57b3cf6bfc5c4374c09d7 and it failed!
it was that stupid binary file. I don't know how but the list item was committed in a corrupted state :-(

I'm re-pushing with the none corrupt file.
Sorry about that, that was really weird.

**Comment by follower46** [comment]
Created https://github.com/clockworkpi/launcher/pull/68 with the corrected file.
I have no idea how that happened :-(
